src/components/ButtonPanel.js

```diff
@@ -1,38 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-export default function ButtonPanel() {
+export default function ButtonPanel({ clickHandler }) {
   return (
     <div className="ButtonPanel">
       <div>
-        <Button buttonName="AC" />
-        <Button buttonName="+/-" />
-        <Button buttonName="%" />
-        <Button buttonName="÷" />
+        <Button buttonName="AC" clickHandler={() => clickHandler('AC')} />
```

Comment:
[Optional]

You could use props instead of repeating yourself.

Example;

`const { clickHandler } = props`

`<Button buttonName="+/-" clickHandler= {clickHandler}  />`